.gitignore

```diff
@@ -18,6 +18,8 @@ src/olympia/lib/product_json/*.json.last_modified
 src/olympia/lib/product_json/regions/*.json
 src/olympia/lib/product_json/regions/.last_update
 src/olympia/lib/product_json/regions/*.json.last_modified
+src/olympia/files/fixtures/files/recurse.zip.xpi
```

Comment:
These are created by the tests, I'd love to clean them up but didn't know if it mattered as the tests sometimes make other files in that `fixtures` folder too. Let me know if this is bad.